lstchain/reco/r0_to_dl1.py

```diff
@@ -510,10 +526,27 @@ def r0_to_dl1(
                                            calibration_index,
                                            event.mon.tel[tel_id],
                                            new_ped=new_ped_event, new_ff=new_ff_event)
-
-                    # calibrate and gain select the event by hand for DL1
+                    
+                    # write the calibrated R1 waveform without gain selection
+                    source.r0_r1_calibrator.select_gain = False
                     source.r0_r1_calibrator.calibrate(event)
-
+                
+                    if interleaved_writer is not None:
+                        interleaved_writer(event)
+
+                    # gain select the events
+                    source.r0_r1_calibrator.select_gain = True
```

Comment:
This is not needed any more

Reply:
yes, we need it, because I above I had to put it to False...